book/team.yaml

```diff
@@ -203,4 +203,40 @@ people:
         link: https://github.com/jomey
     user_groups:
       - Technology Specialist
+  # =============
+  - title: Dylan Boyd
+  avatar: https://avatars.githubusercontent.com/u/28358098?v=4
+  role: Tutorial Lead
+  organizations:
```

Comment:
thanks @db1950! indentation levels matter for formatting in this file, please match the indentation against earlier entries

Reply:
Hi @scottyhq. Looks like it's fixed now. I apologize!